### PR TITLE
Raster layer file

### PR DESCRIPTION
**Pull request by saratchandrakarumuri**

### CODE REVIEW

File: arcgis-runtime-samples-macos/Layers/Raster layer (file)/RasterLayerVC.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        
+        //create raster layer using raster
+        self.rasterLayer = AGSRasterLayer(raster: raster)
+        
+        //initialize map with raster layer as the basemap
+        self.map = AGSMap(basemap: AGSBasemap.imagery())
+        
+        self.mapView.map = map
+        
+        self.mapView.map?.operationalLayers.add(rasterLayer!)
```
Comment (dg0yal) [inline arcgis-runtime-samples-macos/Layers/Raster layer (file)/RasterLayerVC.swift]: add comment

File: arcgis-runtime-samples-macos/Layers/Raster layer (file)/RasterLayerVC.swift [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        
+        self.rasterLayer.load { (error) in
```
Comment (dg0yal) [inline arcgis-runtime-samples-macos/Layers/Raster layer (file)/RasterLayerVC.swift]: add comment

File: arcgis-runtime-samples-macos/Layers/Raster layer (file)/RasterLayerVC.swift [hunk 3]
```diff
@@ -0,0 +1,41 @@
+//
+//  RasterLayerVC.swift
+//  arcgis-runtime-samples-macos
+//
+//  Created by Sarat Karumuri on 6/5/17.
```
Comment (dg0yal) [inline arcgis-runtime-samples-macos/Layers/Raster layer (file)/RasterLayerVC.swift]: copyright needs to be changes

File: arcgis-runtime-samples-macos/Layers/Raster layer (file)/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Raster layer (file)
+
+This sample demonstrates how to create and use a raster layer made from a local raster file.
+
+![](image1.png)
+
+## How it works
+
+`AGSRasterLayer` provides an initializer called `init(raster:)` that takes a `AGSRaster`.
```
Comment (dg0yal) [inline arcgis-runtime-samples-macos/Layers/Raster layer (file)/README.md]: same change as suggested in ios sample

**Comment by saratchandrakarumuri** [comment]
@dg0yal Made changes as per your feedback

File: arcgis-runtime-samples-macos/Layers/Raster layer (file)/RasterLayerViewController.swift [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        //assign map to the map view
+        self.mapView.map = map
+        
+        //add the raster layer to the operational layers of the map
+        self.mapView.map?.operationalLayers.add(rasterLayer!)
+        
+        //set map view's viewpoint to the raster layer's full extent
+        self.rasterLayer.load { [weak self] (error) in
+            guard error == nil else {
+                self?.showAlert(messageText: "Error", informativeText: "Error while loading raster layer :: \(error?.localizedDescription)")
```
Comment (singhgag) [inline arcgis-runtime-samples-macos/Layers/Raster layer (file)/RasterLayerViewController.swift]: implicitly unwrap the `error`

**Comment by singhgag** [comment]
@saratchandrakarumuri Just one change. Otherwise the sample looks good.

**Comment by saratchandrakarumuri** [comment]
@singhgag Implicitly unwrapped error

**Pull request merged**